packages/@o3r/schematics/src/rule-factories/ng-add/dependencies.ts

```diff
@@ -71,6 +71,8 @@ export interface SetupDependenciesOptions {
   runAfterTasks?: TaskId[];
   /** Callback to run after the task ID is calculated */
   scheduleTaskCallback?: (taskIds?: TaskId[]) => void;
+  /** Working directory */
+  workingDirectory?: string;
```

Comment:
maybe you should indicate that this is for the installation process only (as the package.json edited is not affected by this parameter)